students/minwookjang/users/views.py

```diff
@@ -50,9 +53,12 @@ def post(self, request):
         data = json.loads(request.body)
 
         try:
-            if not User.objects.filter(email = data['email'], password = data['password']):
+            if not User.objects.filter(email = data['email']):
```

Comment:
존재 유무를 판단할 경우 exists() method을 사용해 주세요. exists는 존재 여부만 판단하여 boolean을 반환해 줍니다. 단순히 filter만 사용할 경우 불필요한 데이터들까지 가져오게 되어서 비효율적입니다.
```suggestion
            if not User.objects.filter(email = data['email']).exists():
```